### PR TITLE
Test feeds that have a `site.lang`

### PR DESCRIPTION
**Pull request by pathawks**
See #162 #163

### CODE REVIEW

File: spec/jekyll-feed_spec.rb [hunk 1]
```diff
@@ -267,12 +282,4 @@
       expect(contents).to include('<?xml-stylesheet type="text/xml" href="http://example.org/feed.xslt.xml"?>')
     end
   end
-
-  context "with site.lang set" do
-    let(:overrides) { { "lang" => "en-US" } }
-
-    it "should set the language" do
-      expect(contents).to match %r{type="text/html" hreflang="en-US" />}
```
Comment (pathawks) [inline spec/jekyll-feed_spec.rb]: Perhaps I should leave this test, as it appears to be checking [something else](https://github.com/jekyll/jekyll-feed/blob/v0.9.1/lib/jekyll-feed/feed.xml#L8)

**Comment by ashmaroli** [comment]
There's one other place in the [feed template](https://github.com/jekyll/jekyll-feed/blob/master/lib/jekyll-feed/feed.xml#L36) where things could break again..

File: spec/jekyll-feed_spec.rb (hunk 1 above)
Comment (ashmaroli) [inline spec/jekyll-feed_spec.rb]: yes, this should be left to test that parameter

Comment (pathawks) [inline spec/jekyll-feed_spec.rb]: How does [this look](https://github.com/jekyll/jekyll-feed/pull/164/files#diff-46663446120a8d6231c2e8f9f752dcfcR160) as a replacement?

Comment (ashmaroli) [inline spec/jekyll-feed_spec.rb]: Better.. and more complete :+1:

**Comment by ashmaroli** [comment]
Because [this line](https://github.com/jekyll/jekyll-feed/blob/master/lib/jekyll-feed/feed.xml#L36) in the template could output to:
```xml
<entryxml:lang="en-us">
```
based on the recent bug, I think that should be edited on the `master` branch

**Comment by ashmaroli** [comment]
```ruby
    context "with post.lang set" do
      it "renders the feed meta" do
         expected = %r!<entry xml:lang="en-us">!
         expect(contents).to match(expected)
       end
    end
```

**Comment by pathawks** [comment]
Isn't that what [this](https://github.com/jekyll/jekyll-feed/pull/164/files#diff-46663446120a8d6231c2e8f9f752dcfcR157) is testing?

**Comment by ashmaroli** [comment]
> Isn't that what this is testing?

does not test the entire stretch of the markup.. the missing whitespace remained undetected..

File: spec/jekyll-feed_spec.rb [hunk 2]
```diff
@@ -136,6 +138,31 @@
...
+      it "outputs the correct language" do
+        expect(feed.lang).to eql(lang)
+      end
+
+      it "sets the language of entries" do
+        post = feed.items.first
+        expect(post.lang).to eql(lang)
+      end
+
+      it "renders the feed meta" do
```
Comment (pathawks) [inline spec/jekyll-feed_spec.rb]: This is testing a link in the feed to the site; not the feed meta tag. Perhaps it could use a better name.

**Comment by pathawks** [comment]
@jekyllbot: merge +dev

**Pull request merged**